frontend/server/src/Controllers/Problem.php

```diff
@@ -4790,7 +4790,7 @@ public static function getProblemListForSmarty(
         }
 
         // Defaults for offset and pageSize
-        $offset = $r->ensureOptionalInt('offset') ?? 0;
+        $offset = $r->ensureOptionalInt('offset') ?? null;
```

Comment:
Las funciones de tipo `ensureOptionalXxx` ya regresan por si solas `null` en caso de no ser requerido el parámetro:
```suggestion
        $offset = $r->ensureOptionalInt('offset');
```
Mismo caso para los demás cambios